frontend/src/components/table/Table.tsx

```diff
@@ -0,0 +1,34 @@
+import { Table as ATable, TableProps } from 'antd';
+import React from 'react';
+import styled from 'styled-components';
+import { size } from '../../constants';
+
+import { Colors } from '../../styles';
+
+const StyledTable = styled(ATable)`
+    box-shadow: 0px 5px 30px -6px rgba(100, 100, 100, 0.5);
+    overflow-x: auto;
+    background: ${Colors.onPrimary};
+
+    .ant-table-thead > tr > th {
+        font-weight: 400;
+        color: ${Colors.gray};
```

Comment:
`Colors.gray` nie jest odpowiednią nazwą w tym przypadku. Co jeśli będziemy chcieli zmienić kolorystykę aplikacji i dać np. czerwone napisy? Wtedy musielibyśmy przejść przez wszystkie miejsca w kodzie i podmienić wartości np. z `Colors.gray` na `Colors.red`. Natomiast jeśli korzystać będziemy z bardziej generycznych nazw, to wtedy wystarczy zmiana w jednym miejscu (plik colors.ts). Tutaj możesz zmienić nazwę na `Colors.text`